js_modules/dagit/packages/ui/src/components/Dialog.tsx

```diff
@@ -61,6 +61,26 @@ export const DialogBody: React.FC = (props) => {
   );
 };
 
+// The subheadings used in the backfill modals
+export const DialogBodySection = ({
+  title,
+  children,
+}: {
+  title: string | React.ReactNode;
+  children: React.ReactNode;
+}) => (
+  <Box flex={{direction: 'column', gap: 8}}>
+    <strong style={{display: 'block'}}>{title}</strong>
```

Comment:
Possibly a styled `h3` or `h4` or something instead of a `strong`?

Reply:
Hmmm good call. It looks like this is equivalent to `Subheading` except with a slightly different line height, will use that for consistency